Makefiles/sledconf.esp32

```diff
@@ -53,7 +53,5 @@ CFLAGS += -I${PLATFORMIO_HOME}/packages/framework-arduinoespressif32/tools/sdk/i
 CFLAGS += -I${PLATFORMIO_HOME}/packages/framework-arduinoespressif32/tools/sdk/include/driver/
 CFLAGS += -I${PLATFORMIO_HOME}/packages/framework-arduinoespressif32/tools/sdk/include/esp_ringbuf/
 
-CC := ${PLATFORMIO_HOME}/packages/toolchain-xtensa32/bin/xtensa-esp32-elf-cc
-AR := ${PLATFORMIO_HOME}/packages/toolchain-xtensa32/bin/xtensa-esp32-elf-ar
-AR := xtensa-esp32-elf-ar
-
+# led lib for out mod
+CFLAGS += -I${PLATFORMIO_HOME}/lib/ESP32-Digital-RGB-LED-Drivers/include/
```

Comment:
You can actually do this a bit better I believe.

You can try adding a `.incs` file with that `-I${PLATFORMIO_HOME}/lib/ESP32-Digital-RGB-LED-Drivers/include/`, it should propagate. Not sure if Make exports those variables, though.

Reply:
Yes that would be nice but I did some tests and it looks like it won't get exported?

Reply:
That's weird, it should.